index.html

```diff
@@ -26,17 +26,18 @@
         <script type="text/javascript" src="javascript/create_retirement_graph_module.js"></script>
         <script type="text/javascript" src="javascript/app.js"></script>
 
-        <title>Financial Freedom</title>
+        <title>Plenti: Financial freedom for all</title>
     </head>
     
     <body ng-controller="bodyController">
         <div class="header-wrapper" ng-controller="HeaderController">
-            <div id="header-accent"></div>
-            <nav class="navbar navbar-default navbar-custom">
+            <nav class="navbar navbar-default navbar-fixed-top navbar-custom">
                 <div class="container-fluid">
                     <div class="navbar-header">
-                        <a class="header-text navbar-brand" id="product-name" ng-click="goToRoute('/income')">Rung</a>
+                        <a class="header-text navbar-brand" id="product-name" ng-click="goToRoute('/income')"><img src="/img/bowl.png" class="brand-logo"></a>
+                        <a class="header-text navbar-brand" id="product-name" ng-click="goToRoute('/income')"><span>Plenti</span></a>

```

Comment:
rather than making the image and the text be wrapped in separate `<a>` tags, wrap both of them in the same tag so we're not duplicating code here. Also, ids have to be 100% unique on the page, both tags here have id="product-name" which can cause issues and is generally bad practice.